epoxy-sample/src/main/java/com/airbnb/epoxy/sample/models/ColorModel.java

```diff
@@ -54,6 +61,31 @@ public void bindChanges(@NonNull ColorHolder holder,
     }
   }
 
+  @Nullable
+  @Override
+  public EpoxyItemAnimation bindChangesAnimated(@NonNull ColorHolder view,
+                                                @NonNull EpoxyModel<?> previouslyBoundModel,
+                                                @Nullable EpoxyItemAnimation interruptedAnimation) {
+
+    int currentColor = ((ColorModel) previouslyBoundModel).color;
+    float currentRotation = 0f;
+
+    if(interruptedAnimation != null) {
+      AnimatorSet previousAnim = (AnimatorSet) ((EpoxyItemAnimation.AnimatorWrapper) interruptedAnimation).wrappedAnimator;
```

Comment:
It does seem true that you could get previous state off the viewholder, the convenience of the previous model seems nice though.

The other coordination work done by the item animator and RV seems useful, but not necessary as you point out. I don't understand enough about recyclerview animations to know what the other downsides to managing animations manually would be - I'd be interested to hear yigit's take on it.

Still, if we can handle all these little things in an easy way so that people can set up animations fluently in their models then that would be a nice addition to the library.